src/ui/public/kuery/index.js

```diff
@@ -17,6 +17,4 @@
  * under the License.
  */
 
-export * from './ast';
-export * from './filter_migration';
-export * from './node_types';
+export * from '../../../utils/kuery';
```

Comment:
Can we just delete this module now?

Reply:
Same comment as above, we could, but we'd have to update all of the references, which is fine by me if you think it's best.